src/main/java/net/minestom/server/instance/AnvilLoader.java

```diff
@@ -361,6 +361,10 @@ private BlockState getBlockState(final Block block) {
         return blockStateId2ObjectCacheTLS.get().computeIfAbsent(block.stateId(), _unused -> new BlockState(block.name(), block.properties()));
     }
 
+    public Path getPath() {
```

Comment:
@NotNull

Reply:
But normal world files would still be supported. 
What if I change this to a nullable function? If you use the normal world files, it return a path. But if you use a non-filesystem world, it returns null.

Reply:
Nullable + a javadoc stating that it is implementation dependent (even though it is not right now) would be an improvement IMO. But can also wait to see what others say.

Reply:
The anvil loader extends the IChunkLoader. Maybe it is better to create another class that extends the IChunkLoader too. That class can use the non-filesystem. (if you want to make a new chunkloader)